src/components/Signin.jsx

```diff
@@ -67,7 +67,7 @@ const AuthLink = styled.a`
   i{
```

Comment:
Give icons inline-block property. And apply padding right instead of margin.

Reply:
padding-right , i dont think it would work, padding right would be inside the icon, we need to maintain gap between the icon and content, so margin-right is the correct one.